.changeset/giant-flies-roll.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-sonarqube': patch
+---
+
+Support Sonarqube project keys containing special characters like colons.
```

Comment:
```suggestion
Support SonarQube project keys containing special characters like colons.
```

I could imaging that vale likes it that way and the build will be green.

Reply:
It worked.